compiler/luci/profile/include/luci/Profile/CircleOriginData.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_PROFILE_CIRCLE_ORIGIN_DATA_H__
+#define __LUCI_PROFILE_CIRCLE_ORIGIN_DATA_H__
+
+#include <string>
+
+namespace luci
+{
+
+class CircleOriginData
```

Comment:
Can we rename this to `CircleOrigin` ?

Reply:
At first, I thought `CircleOriginInfo`, `CircleOriginData`, `CircleOrigin` and so on.
But I excluded `CircleOrigin` because the word is very similar with `CircleNodeOrigin` and thus it causes confusion :(